sdk/test/opentelemetry/sdk/trace/export/in_memory_span_exporter_test.rb

```diff
@@ -7,4 +7,67 @@
 require 'test_helper'
 
 describe OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter do
+  export = OpenTelemetry::SDK::Trace::Export
```

Comment:
Two questions:
1. Do we need to do this? We only access this in one place AFAICT (`let(:subject) { ... }`), so it doesn’t seem worthwhile.
1. In other places, we’ve done this “trick” to simplify access to the class under test, which in this case would look like:
```ruby
InMemorySpanExporter = OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter
```

Reply:
We're also using it to shorten the error constants ```OpenTelemetry::SDK::Trace::Export::SUCCESS``` and ```OpenTelemetry::SDK::Trace::Export::FAILED_NOT_RETRYABLE```.

I'm avoiding using Constants for the alias names, because we end up with multiple definitions warnings if multiple test files take the same approach.

I'm not wedded to the short names -- I was just trying to keep line length under control. I can swap them back out for the fully qualified versions if you prefer.

Reply:
Ah, missed the error constants. I don't have a strong opinion.